lib/spack/spack/test/environment_modifications.py

```diff
@@ -158,75 +210,96 @@ def test_unset(env):
         os.environ["UNSET_ME"]
 
 
-@pytest.mark.not_on_windows("Not supported on Windows (yet)")
-def test_filter_system_paths(miscellaneous_paths):
+def test_filter_system_paths(system_paths_for_os, non_system_paths_for_os):
     """Tests that the filtering of system paths works as expected."""
-    filtered = filter_system_paths(miscellaneous_paths)
-    expected = [
-        "/usr/local/Cellar/gcc/5.3.0/lib",
-        "/usr/local/opt/some-package/lib",
-        "/usr/opt/lib",
-        "/opt/some-package/include",
-        "/opt/some-package/local/..",
-    ]
-    assert filtered == expected
+    filtered = filter_system_paths(system_paths_for_os + non_system_paths_for_os)
+    assert filtered == non_system_paths_for_os
 
 
-# TODO 27021
-@pytest.mark.not_on_windows("Not supported on Windows (yet)")
-def test_set_path(env):
+@pytest.mark.parametrize("name,elements,separator", [("A", ["foo", "bar", "baz"], os.pathsep)])
```

Comment:
This parameterization has a single instance - IMO there should be a comment (about what kinds of parameterizations would make sense to add) or this should not be parameterized.

Reply:
Could not find a reason this was parameterized nor could I think of more parameterizations, removed the parameterization.